dotnetv3/StepFunctions/Actions/StepFunctionsWrapper.cs

```diff
@@ -0,0 +1,283 @@
+    {
+        var request = new CreateStateMachineRequest
+        {
+            Name = stateMachineName,
+            Definition = definition,
+            RoleArn = roleArn
+        };
+
+        var response =
+            await _amazonStepFunctions.CreateStateMachineAsync(request);
+        return response.StateMachineArn;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.CreateStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.DeleteActivity]
+    /// <summary>
+    /// Delete a Step Machine activity.
+    /// </summary>
+    /// <param name="activityArn">The Amazon Resource Name (ARN) of
+    /// the activity.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> DeleteActivity(string activityArn)
+    {
+        var response = await _amazonStepFunctions.DeleteActivityAsync(new DeleteActivityRequest { ActivityArn = activityArn });
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteActivity]
+
+    // snippet-start:[StepFunctions.dotnetv3.DeleteStateMachine]
+    /// <summary>
+    /// Delete a Step Functions state machine.
+    /// </summary>
+    /// <param name="stateMachineArn">The Amazon Resource Name (ARN) of the
+    /// state machine.</param>
+    /// <returns>A Boolean value indicating the success of the action.</returns>
+    public async Task<bool> DeleteStateMachine(string stateMachineArn)
+    {
+        var response = await _amazonStepFunctions.DeleteStateMachineAsync(new DeleteStateMachineRequest
+        { StateMachineArn = stateMachineArn });
+        return response.HttpStatusCode == HttpStatusCode.OK;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DeleteStateMachine]
+
+    // snippet-start:[StepFunctions.dotnetv3.DescribeExecution]
+    /// <summary>
+    /// Retrieve information about the specified Step Functions execution.
+    /// </summary>
+    /// <param name="executionArn">The Amazon Resource Name (ARN) of the
+    /// Step machines execution.</param>
+    /// <returns>The API response returned by the API.</returns>
+    public async Task<DescribeExecutionResponse> DescribeExecutionAsync(string executionArn)
+    {
+        var response = await _amazonStepFunctions.DescribeExecutionAsync(new DescribeExecutionRequest { ExecutionArn = executionArn });
+        return response;
+    }
+
+    // snippet-end:[StepFunctions.dotnetv3.DescribeExecution]
+
+    // snippet-start:[StepFunctions.dotnetv3.DescribeStateMachine]
+    /// <summary>
+    /// Retrieve information about the specified Step Functions state machine.
+    /// </summary>
+    /// <param name="StateMachineArn">The Amazon Resource Name (ARN) of the
+    /// Step Functions state machine to retrieve.</param>
+    /// <returns>Information about the specified Step Functions state machine.</returns>
+    public async Task<DescribeStateMachineResponse> DescribeStateMachineAsync(string StateMachineArn)
```

Comment:
Could not find this method in the scenario or the tests.

Reply:
Added in both places. This call was added to the modified scenario.